packages/expo-clipboard/ios/ClipboardModule.swift

```diff
@@ -11,17 +12,28 @@ public class ClipboardModule: Module {
 
     // MARK: Strings
 
-    function("getStringAsync") { () -> String in
-      return UIPasteboard.general.string ?? ""
+    function("getStringAsync") { (options: GetStringOptions) -> String in
+      switch options.preferredFormat {
+      case .plainText:
+        return UIPasteboard.general.string ?? ""
+      case .html:
+        return UIPasteboard.general.html ?? ""
+      }
     }
 
-    function("setStringAsync") { (content: String?) -> Bool in
-      UIPasteboard.general.string = content ?? ""
+    function("setStringAsync") { (content: String?, options: SetStringOptions) -> Bool in
+      switch options.inputFormat {
+      case .plainText:
+        UIPasteboard.general.string = content
+      case .html:
+        UIPasteboard.general.html = content
+      }
+
       return true
     }
 
     function("hasStringAsync") { () -> Bool in
-      return UIPasteboard.general.hasStrings
+      UIPasteboard.general.hasStrings || UIPasteboard.general.contains(pasteboardTypes: [kUTTypeHTML as String, kUTTypeRTF as String])
```

Comment:
I would recommend always using explicit `return` for clarity. When the function body is a one-liner, then maybe it's ok to skip `return` as it doesn't lose its legibility, but on the other side it seems better to have a consistent and simple rule for this. What do you think?

```suggestion
      return UIPasteboard.general.hasStrings || UIPasteboard.general.contains(pasteboardTypes: [kUTTypeHTML as String, kUTTypeRTF as String])
```

Reply:
I'm for using explicit `return` always.

But how would you split this line? It's "only" 142 chars long, I know warn/error limit is 160/200, but that's too long IMO
```suggestion
      UIPasteboard.general.hasStrings ||
        UIPasteboard.general.contains(pasteboardTypes: [kUTTypeHTML as String, kUTTypeRTF as String])
```
^ like this _(Kotlin lint would do it this way)_ or some other way?

Reply:
Yeah, this suggestion makes sense. By the way, it's following Google's style guide 😉 https://google.github.io/swift/#line-wrapping